### PR TITLE
Add german language-file to Terra-Station

### PR DESCRIPTION
**Pull request by BlockJayn**
Translated and proven by the German LUNATICs-Community

### CODE REVIEW

**Comment by BlockJayn** [comment]
If you need any further words/descriptions to be translated to German just let me know, I'll do that

File: src/locales/it.json [hunk 1]
```diff
@@ -1,217 +0,0 @@
-{
```
Comment (rafaelzimmermann) [inline src/locales/it.json:1]: Looks like you deleted it.json by mistake.

**Review by rafaelzimmermann — COMMENTED** [review]
To make the language available in the selector you also need to adjust this file: src/config/lang.ts

File: src/locales/it.json (hunk 1 above)
Comment (BlockJayn) [reply src/locales/it.json:1]: Oops sorry, definitely a mistake, didn't noticed that! Unfortunately I don't know how to fix that mistake, it's my first time using github. I hope you can help me out here :)

Comment (BlockJayn) [reply src/locales/it.json:1]: > To make the language available in the selector you also need to adjust this file: src/config/lang.ts

@rafaelzimmermann As requested, to make german language available "de" has been added in line 8 and 28.
**Link:** https://github.com/terra-money/station/pull/37

If anything else is needed, just let me know!

**Comment by simcheolhwan** [comment]
You removed `it.json` file.

**Comment by simcheolhwan** [comment]
Add `de.json` to #37 please

File: src/locales/it.json (hunk 1 above)
Comment (rafaelzimmermann) [reply src/locales/it.json:1]: To make the language available in the selector you also need to adjust this file: src/config/lang.ts

**Pull request closed without merging**